sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -10,26 +10,25 @@
  */
 public final class DirectConnectionConfig {
     //  Constants
-    private static final Duration DEFAULT_IDLE_CHANNEL_TIMEOUT = Duration.ZERO;
     private static final Duration DEFAULT_IDLE_ENDPOINT_TIMEOUT = Duration.ofSeconds(70L);
-    private static final int DEFAULT_MAX_CHANNELS_PER_ENDPOINT = 30;
+    private static final int DEFAULT_MAX_CONNECTIONS_PER_ENDPOINT = 30;
     private static final int DEFAULT_MAX_REQUESTS_PER_ENDPOINT = 10;
 
     private Duration connectionTimeout;
-    private Duration idleChannelTimeout;
+    private Duration idleConnectionTimeout;
     private Duration idleEndpointTimeout;
-    private int maxChannelsPerEndpoint;
-    private int maxRequestsPerChannel;
+    private int maxConnectionsPerEndpoint;
+    private int maxRequestsPerConnection;
 
     /**
      * Constructor.
      */
     public DirectConnectionConfig() {
         this.connectionTimeout = null;
```

Comment:
```java
this.connectionTimeout = null;	
```
why is this null?

Reply:
will get rid of the redundant initializer.